### ISSUE TITLE
Unable to build after installing cordova-plugin-android-fingerprint-auth plugin

### ISSUE THREAD

**Issue by bittusinghintruder** [mjwheatley/cordova-plugin-android-fingerprint-auth#71]
I am unable to build if i install the cordova-plugin-android-fingerprint-auth plugin for android. If i remove the plugin build got success.
Output log for the reference:
`ANDROID_HOME=C:\ANDROID\sdk
JAVA_HOME=C:\Program Files\Java\jdk1.8.0_102
Running: D:\Practice codes\android-fingerPrint -ionic\androidFingureTouch\platforms\android\gradlew cdvBuildDebug -b D:\Practice codes\android-fingerPrint -ionic\androidFingureTouch\p
latforms\android\build.gradle -Dorg.gradle.daemon=true
:preBuild
:compileDebugNdk
:preDebugBuild
:checkDebugManifest
:CordovaLib:compileLint
:CordovaLib:copyDebugLint UP-TO-DATE
:CordovaLib:mergeDebugProguardFiles
:CordovaLib:preBuild
:CordovaLib:preDebugBuild
:CordovaLib:checkDebugManifest
:CordovaLib:prepareDebugDependencies
:CordovaLib:compileDebugAidl
:CordovaLib:compileDebugRenderscript
:CordovaLib:generateDebugBuildConfig
:CordovaLib:generateDebugAssets UP-TO-DATE
:CordovaLib:mergeDebugAssets
:CordovaLib:generateDebugResValues
:CordovaLib:generateDebugResources
:CordovaLib:packageDebugResources
:CordovaLib:processDebugManifest
:CordovaLib:processDebugResources
:CordovaLib:generateDebugSources
:CordovaLib:compileDebugJavaNote: Some input files use or override a deprecated API.
Note: Recompile with -Xlint:deprecation for details.

:CordovaLib:processDebugJavaRes UP-TO-DATE
:CordovaLib:packageDebugJar
:CordovaLib:compileDebugNdk
:CordovaLib:packageDebugJniLibs UP-TO-DATE
:CordovaLib:packageDebugLocalJar UP-TO-DATE
:CordovaLib:packageDebugRenderscript UP-TO-DATE
:CordovaLib:bundleDebug
:prepareAndroidCordovaLibUnspecifiedDebugLibrary
:preReleaseBuild
:CordovaLib:copyReleaseLint UP-TO-DATE
:CordovaLib:mergeReleaseProguardFiles
:CordovaLib:preReleaseBuild
:CordovaLib:checkReleaseManifest
:CordovaLib:prepareReleaseDependencies
:CordovaLib:compileReleaseAidl
:CordovaLib:compileReleaseRenderscript
:CordovaLib:generateReleaseBuildConfig
:CordovaLib:generateReleaseAssets UP-TO-DATE
:CordovaLib:mergeReleaseAssets
:CordovaLib:generateReleaseResValues
:CordovaLib:generateReleaseResources
:CordovaLib:packageReleaseResources
:CordovaLib:processReleaseManifest
:CordovaLib:processReleaseResources
:CordovaLib:generateReleaseSources
:CordovaLib:compileReleaseJavaNote: Some input files use or override a deprecated API.

:CordovaLib:processReleaseJavaRes UP-TO-DATE
:CordovaLib:packageReleaseJar
:CordovaLib:compileReleaseNdk
:CordovaLib:packageReleaseJniLibs UP-TO-DATE
:CordovaLib:packageReleaseLocalJar UP-TO-DATE
:CordovaLib:packageReleaseRenderscript UP-TO-DATE
:CordovaLib:bundleReleaseNote: Recompile with -Xlint:deprecation for details.

:prepareComAndroidSupportSupportCompat2500Library
:prepareComAndroidSupportSupportCoreUi2500Library
:prepareComAndroidSupportSupportCoreUtils2500Library
:prepareComAndroidSupportSupportFragment2500Library
:prepareComAndroidSupportSupportMediaCompat2500Library
:prepareComAndroidSupportSupportV42500Library
:prepareDebugDependencies
:compileDebugAidl
:compileDebugRenderscript
:generateDebugBuildConfig
:generateDebugAssets UP-TO-DATE
:mergeDebugAssets
:generateDebugResValues
:generateDebugResources
:mergeDebugResources
:processDebugManifest
:processDebugResources
:generateDebugSources
:compileDebugJavaD:\Practice codes\android-fingerPrint -ionic\androidFingureTouch\platforms\android\src\com\cordova\plugin\android\fingerprintauth\FingerprintAuth.java:17: error: pack
age android.hardware.fingerprint does not exist
import android.hardware.fingerprint.FingerprintManager;
                                   ^
D:\Practice codes\android-fingerPrint -ionic\androidFingureTouch\platforms\android\src\com\cordova\plugin\android\fingerprintauth\FingerprintAuth.java:18: error: package android.secur
ity.keystore does not exist
import android.security.keystore.KeyGenParameterSpec;
                                ^
D:\Practice codes\android-fingerPrint -ionic\androidFingureTouch\platforms\android\src\com\cordova\plugin\android\fingerprintauth\FingerprintAuth.java:19: error: package android.secur
ity.keystore does not exist
import android.security.keystore.KeyProperties;
                                ^
D:\Practice codes\android-fingerPrint -ionic\androidFingureTouch\platforms\android\src\com\cordova\plugin\android\fingerprintauth\FingerprintAuthenticationDialogFragment.java:25: erro
r: package android.hardware.fingerprint does not exist
import android.hardware.fingerprint.FingerprintManager;
                                   ^
D:\Practice codes\android-fingerPrint -ionic\androidFingureTouch\platforms\android\src\com\cordova\plugin\android\fingerprintauth\FingerprintUiHelper.java:21: error: package android.h
ardware.fingerprint does not exist
import android.hardware.fingerprint.FingerprintManager;
                                   ^
D:\Practice codes\android-fingerPrint -ionic\androidFingureTouch\platforms\android\src\com\cordova\plugin\android\fingerprintauth\FingerprintUiHelper.java:31: error: package Fingerpri
ntManager does not exist
public class FingerprintUiHelper extends FingerprintManager.AuthenticationCallback {
                                                           ^
D:\Practice codes\android-fingerPrint -ionic\androidFingureTouch\platforms\android\src\com\cordova\plugin\android\fingerprintauth\FingerprintAuth.java:67: error: cannot find symbol
    private FingerprintManager mFingerPrintManager;
            ^
  symbol:   class FingerprintManager
  location: class FingerprintAuth
D:\Practice codes\android-fingerPrint -ionic\androidFingureTouch\platforms\android\src\com\cordova\plugin\android\fingerprintauth\FingerprintAuth.java:578: error: package FingerprintM
anager does not exist
                                       FingerprintManager.AuthenticationResult result) {
                                                         ^
D:\Practice codes\android-fingerPrint -ionic\androidFingureTouch\platforms\android\src\com\cordova\plugin\android\fingerprintauth\FingerprintAuthenticationDialogFragment.java:54: erro
r: package FingerprintManager does not exist
    private FingerprintManager.CryptoObject mCryptoObject;
                              ^
D:\Practice codes\android-fingerPrint -ionic\androidFingureTouch\platforms\android\src\com\cordova\plugin\android\fingerprintauth\FingerprintAuthenticationDialogFragment.java:181: err
or: package FingerprintManager does not exist
    public void setCryptoObject(FingerprintManager.CryptoObject cryptoObject) {
                                                  ^
D:\Practice codes\android-fingerPrint -ionic\androidFingureTouch\platforms\android\src\com\cordova\plugin\android\fingerprintauth\FingerprintAuthenticationDialogFragment.java:256: err
or: package FingerprintManager does not exist
    public void onAuthenticated(FingerprintManager.AuthenticationResult result) {
                                                  ^
D:\Practice codes\android-fingerPrint -ionic\androidFingureTouch\platforms\android\src\com\cordova\plugin\android\fingerprintauth\FingerprintUiHelper.java:37: error: cannot find symbo
l
    private final FingerprintManager mFingerprintManager;
                  ^
  symbol:   class FingerprintManager
  location: class FingerprintUiHelper
D:\Practice codes\android-fingerPrint -ionic\androidFingureTouch\platforms\android\src\com\cordova\plugin\android\fingerprintauth\FingerprintUiHelper.java:43: error: package Fingerpri
ntManager does not exist
    private static FingerprintManager.AuthenticationResult fingerprintResult;
                                     ^
D:\Practice codes\android-fingerPrint -ionic\androidFingureTouch\platforms\android\src\com\cordova\plugin\android\fingerprintauth\FingerprintUiHelper.java:70: error: cannot find symbo
l
    private FingerprintUiHelper(Context context, FingerprintManager fingerprintManager,
                                                 ^
  symbol:   class FingerprintManager
  location: class FingerprintUiHelper
D:\Practice codes\android-fingerPrint -ionic\androidFingureTouch\platforms\android\src\com\cordova\plugin\android\fingerprintauth\FingerprintUiHelper.java:84: error: package Fingerpri
ntManager does not exist
    public void startListening(FingerprintManager.CryptoObject cryptoObject) {
                                                 ^
D:\Practice codes\android-fingerPrint -ionic\androidFingureTouch\platforms\android\src\com\cordova\plugin\android\fingerprintauth\FingerprintUiHelper.java:150: error: package Fingerpr
intManager does not exist
    public void onAuthenticationSucceeded(FingerprintManager.AuthenticationResult result) {
                                                            ^
D:\Practice codes\android-fingerPrint -ionic\androidFingureTouch\platforms\android\src\com\cordova\plugin\android\fingerprintauth\FingerprintUiHelper.java:204: error: package Fingerpr
intManager does not exist
        void onAuthenticated(FingerprintManager.AuthenticationResult result);
                                               ^
D:\Practice codes\android-fingerPrint -ionic\androidFingureTouch\platforms\android\src\com\cordova\plugin\android\fingerprintauth\FingerprintUiHelper.java:52: error: cannot find symbo
l
        private final FingerprintManager mFingerPrintManager;
                      ^
  symbol:   class FingerprintManager
  location: class FingerprintUiHelperBuilder
D:\Practice codes\android-fingerPrint -ionic\androidFingureTouch\platforms\android\src\com\cordova\plugin\android\fingerprintauth\FingerprintUiHelper.java:55: error: cannot find symbo
l
        public FingerprintUiHelperBuilder(Context context, FingerprintManager fingerprintManager) {
                                                           ^
  symbol:   class FingerprintManager
  location: class FingerprintUiHelperBuilder
D:\Practice codes\android-fingerPrint -ionic\androidFingureTouch\platforms\android\src\com\cordova\plugin\android\fingerprintauth\FingerprintAuth.java:152: error: no suitable method f
ound for getSystemService(Class<KeyguardManager>)
        mKeyguardManager = cordova.getActivity().getSystemService(KeyguardManager.class);
                                                ^
    method Context.getSystemService(String) is not applicable
      (argument mismatch; Class<KeyguardManager> cannot be converted to String)
    method ContextWrapper.getSystemService(String) is not applicable
      (argument mismatch; Class<KeyguardManager> cannot be converted to String)
    method ContextThemeWrapper.getSystemService(String) is not applicable
      (argument mismatch; Class<KeyguardManager> cannot be converted to String)
    method Activity.getSystemService(String) is not applicable
      (argument mismatch; Class<KeyguardManager> cannot be converted to String)
D:\Practice codes\android-fingerPrint -ionic\androidFingureTouch\platforms\android\src\com\cordova\plugin\android\fingerprintauth\FingerprintAuth.java:154: error: cannot find symbol
                            .getSystemService(FingerprintManager.class);
                                              ^
  symbol:   class FingerprintManager
  location: class FingerprintAuth
D:\Practice codes\android-fingerPrint -ionic\androidFingureTouch\platforms\android\src\com\cordova\plugin\android\fingerprintauth\FingerprintAuth.java:157: error: cannot find symbol
            mKeyGenerator = KeyGenerator.getInstance(KeyProperties.KEY_ALGORITHM_AES, ANDROID_KEY_STORE);
                                                     ^
  symbol:   variable KeyProperties
  location: class FingerprintAuth
D:\Practice codes\android-fingerPrint -ionic\androidFingureTouch\platforms\android\src\com\cordova\plugin\android\fingerprintauth\FingerprintAuth.java:168: error: cannot find symbol
            mCipher = Cipher.getInstance(KeyProperties.KEY_ALGORITHM_AES + "/"
                                         ^
  symbol:   variable KeyProperties
  location: class FingerprintAuth
D:\Practice codes\android-fingerPrint -ionic\androidFingureTouch\platforms\android\src\com\cordova\plugin\android\fingerprintauth\FingerprintAuth.java:169: error: cannot find symbol
                    + KeyProperties.BLOCK_MODE_CBC + "/" + KeyProperties.ENCRYPTION_PADDING_PKCS7);
                      ^
  symbol:   variable KeyProperties
  location: class FingerprintAuth
D:\Practice codes\android-fingerPrint -ionic\androidFingureTouch\platforms\android\src\com\cordova\plugin\android\fingerprintauth\FingerprintAuth.java:169: error: cannot find symbol
                    + KeyProperties.BLOCK_MODE_CBC + "/" + KeyProperties.ENCRYPTION_PADDING_PKCS7);
                                                           ^
  symbol:   variable KeyProperties
  location: class FingerprintAuth
D:\Practice codes\android-fingerPrint -ionic\androidFingureTouch\platforms\android\src\com\cordova\plugin\android\fingerprintauth\FingerprintAuth.java:233: error: cannot find symbol
                    checkAndRequestPermission(Manifest.permission.USE_FINGERPRINT,
                                                                 ^
  symbol:   variable USE_FINGERPRINT
  location: class permission
D:\Practice codes\android-fingerPrint -ionic\androidFingureTouch\platforms\android\src\com\cordova\plugin\android\fingerprintauth\FingerprintAuth.java:322: error: package FingerprintM
anager does not exist
                                        mFragment.setCryptoObject(new FingerprintManager.CryptoObject(mCipher));
                                                                                        ^
D:\Practice codes\android-fingerPrint -ionic\androidFingureTouch\platforms\android\src\com\cordova\plugin\android\fingerprintauth\FingerprintAuth.java:328: error: package FingerprintM
anager does not exist
                                            mFragment.setCryptoObject(new FingerprintManager.CryptoObject(mCipher));
                                                                                            ^
D:\Practice codes\android-fingerPrint -ionic\androidFingureTouch\platforms\android\src\com\cordova\plugin\android\fingerprintauth\FingerprintAuth.java:434: error: method does not over
ride or implement a method from a supertype
    @Override
    ^
D:\Practice codes\android-fingerPrint -ionic\androidFingureTouch\platforms\android\src\com\cordova\plugin\android\fingerprintauth\FingerprintAuth.java:437: error: cannot find symbol
        super.onRequestPermissionResult(requestCode, permissions, grantResults);
             ^
  symbol: method onRequestPermissionResult(int,String[],int[])
D:\Practice codes\android-fingerPrint -ionic\androidFingureTouch\platforms\android\src\com\cordova\plugin\android\fingerprintauth\FingerprintAuth.java:549: error: cannot find symbol
                    .setEncryptionPaddings(KeyProperties.ENCRYPTION_PADDING_PKCS7)
                                           ^
  symbol:   variable KeyProperties
  location: class FingerprintAuth
D:\Practice codes\android-fingerPrint -ionic\androidFingureTouch\platforms\android\src\com\cordova\plugin\android\fingerprintauth\FingerprintAuth.java:547: error: cannot find symbol
                    .setBlockModes(KeyProperties.BLOCK_MODE_CBC)
                                   ^
  symbol:   variable KeyProperties
  location: class FingerprintAuth
D:\Practice codes\android-fingerPrint -ionic\androidFingureTouch\platforms\android\src\com\cordova\plugin\android\fingerprintauth\FingerprintAuth.java:545: error: package KeyGenParame
terSpec does not exist
            mKeyGenerator.init(new KeyGenParameterSpec.Builder(mClientId,
                                                      ^
D:\Practice codes\android-fingerPrint -ionic\androidFingureTouch\platforms\android\src\com\cordova\plugin\android\fingerprintauth\FingerprintAuth.java:546: error: cannot find symbol
                    KeyProperties.PURPOSE_ENCRYPT | KeyProperties.PURPOSE_DECRYPT)
                    ^
  symbol:   variable KeyProperties
  location: class FingerprintAuth
D:\Practice codes\android-fingerPrint -ionic\androidFingureTouch\platforms\android\src\com\cordova\plugin\android\fingerprintauth\FingerprintAuth.java:546: error: cannot find symbol
                    KeyProperties.PURPOSE_ENCRYPT | KeyProperties.PURPOSE_DECRYPT)
                                                    ^
  symbol:   variable KeyProperties
  location: class FingerprintAuth
D:\Practice codes\android-fingerPrint -ionic\androidFingureTouch\platforms\android\src\com\cordova\plugin\android\fingerprintauth\FingerprintAuthenticationDialogFragment.java:69: erro
r: cannot find symbol
        mKeyguardManager = (KeyguardManager) getContext().getSystemService(Context.KEYGUARD_SERVICE);
                                             ^
  symbol:   method getContext()
  location: class FingerprintAuthenticationDialogFragment
D:\Practice codes\android-fingerPrint -ionic\androidFingureTouch\platforms\android\src\com\cordova\plugin\android\fingerprintauth\FingerprintAuthenticationDialogFragment.java:71: erro
r: cannot find symbol
                getContext(), getContext().getSystemService(FingerprintManager.class));
                ^
  symbol:   method getContext()
  location: class FingerprintAuthenticationDialogFragment
D:\Practice codes\android-fingerPrint -ionic\androidFingureTouch\platforms\android\src\com\cordova\plugin\android\fingerprintauth\FingerprintAuthenticationDialogFragment.java:71: erro
r: cannot find symbol
                getContext(), getContext().getSystemService(FingerprintManager.class));
                                                            ^
  symbol:   class FingerprintManager
  location: class FingerprintAuthenticationDialogFragment
D:\Practice codes\android-fingerPrint -ionic\androidFingureTouch\platforms\android\src\com\cordova\plugin\android\fingerprintauth\FingerprintAuthenticationDialogFragment.java:71: erro
r: cannot find symbol
                getContext(), getContext().getSystemService(FingerprintManager.class));
                              ^
  symbol:   method getContext()
  location: class FingerprintAuthenticationDialogFragment
D:\Practice codes\android-fingerPrint -ionic\androidFingureTouch\platforms\android\src\com\cordova\plugin\android\fingerprintauth\FingerprintAuthenticationDialogFragment.java:217: err
or: cannot find symbol
                    Toast.makeText(getContext(),
                                   ^
  symbol:   method getContext()
  location: class FingerprintAuthenticationDialogFragment
D:\Practice codes\android-fingerPrint -ionic\androidFingureTouch\platforms\android\src\com\cordova\plugin\android\fingerprintauth\FingerprintUiHelper.java:106: error: method does not
override or implement a method from a supertype
    @Override
    ^
D:\Practice codes\android-fingerPrint -ionic\androidFingureTouch\platforms\android\src\com\cordova\plugin\android\fingerprintauth\FingerprintUiHelper.java:119: error: method does not
override or implement a method from a supertype
    @Override
    ^
D:\Practice codes\android-fingerPrint -ionic\androidFingureTouch\platforms\android\src\com\cordova\plugin\android\fingerprintauth\FingerprintUiHelper.java:124: error: method does not
override or implement a method from a supertype
    @Override
    ^
D:\Practice codes\android-fingerPrint -ionic\androidFingureTouch\platforms\android\src\com\cordova\plugin\android\fingerprintauth\FingerprintUiHelper.java:149: error: method does not
override or implement a method from a supertype
    @Override
    ^
D:\Practice codes\android-fingerPrint -ionic\androidFingureTouch\platforms\android\src\com\cordova\plugin\android\fingerprintauth\FingerprintUiHelper.java:159: error: method getColor
in class Resources cannot be applied to given types;
                mErrorTextView.getResources().getColor(success_color_id, null));
                                             ^
  required: int
  found: int,<null>
  reason: actual and formal argument lists differ in length
D:\Practice codes\android-fingerPrint -ionic\androidFingureTouch\platforms\android\src\com\cordova\plugin\android\fingerprintauth\FingerprintUiHelper.java:180: error: method getColor
in class Resources cannot be applied to given types;
                mErrorTextView.getResources().getColor(warning_color_id, null));
                                             ^
  required: int
  found: int,<null>
  reason: actual and formal argument lists differ in length
D:\Practice codes\android-fingerPrint -ionic\androidFingureTouch\platforms\android\src\com\cordova\plugin\android\fingerprintauth\FingerprintUiHelper.java:191: error: method getColor
in class Resources cannot be applied to given types;
                    mErrorTextView.getResources().getColor(hint_color_id, null));
                                                 ^
  required: int
  found: int,<null>
  reason: actual and formal argument lists differ in length
Note: D:\Practice codes\android-fingerPrint -ionic\androidFingureTouch\platforms\android\src\org\apache\cordova\splashscreen\SplashScreen.java uses or overrides a deprecated API.
Note: Recompile with -Xlint:deprecation for details.
47 errors

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':compileDebugJava'.
> Compilation failed; see the compiler error output for details.

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.
 FAILED

BUILD FAILED

Total time: 12.921 secs

D:\Practice codes\android-fingerPrint -ionic\androidFingureTouch\platforms\android\cordova\node_modules\q\q.js:126
                    throw e;
                    ^
Error code 1 for command: cmd with args: /s /c ""D:\Practice codes\android-fingerPrint -ionic\androidFingureTouch\platforms\android\gradlew" cdvBuildDebug -b "D:\Practice codes\androi
d-fingerPrint -ionic\androidFingureTouch\platforms\android\build.gradle" -Dorg.gradle.daemon=true"
ERROR building one of the platforms: Error: cmd: Command failed with exit code 1
You may not have the required environment or OS to build this project
Error: cmd: Command failed with exit code 1
    at ChildProcess.whenDone (C:\Program Files\nodejs\node_modules\cordova\node_modules\cordova-lib\src\cordova\superspawn.js:134:23)
    at emitTwo (events.js:87:13)
    at ChildProcess.emit (events.js:172:7)
    at maybeClose (internal/child_process.js:821:16)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:211:5)
`

**Comment by harshvardhan12** [comment]
Is you sdk up to date. It seems like your android sdk has not been updated.

**Comment by bittusinghintruder** [comment]
Its updated to API 25 right now any thing else i need to update

**Comment by harshvardhan12** [comment]
Is the plugin getting successfully installed?
According to the errors you are getting,  it is not able to find the packages and Java classes.

**Comment by bittusinghintruder** [comment]
Yes it get install successfully. Ya looks like some java class reference missing but i dint understand is it missing from plugin or some dependency from Android SDK.

**Comment by bittusinghintruder** [comment]
These are the Android SDK dependency i have installed in my system.
![1](https://cloud.githubusercontent.com/assets/12931813/24944884/9d65d2b6-1f7a-11e7-9556-612bc45d3f17.PNG)
![2](https://cloud.githubusercontent.com/assets/12931813/24944885/9d66d508-1f7a-11e7-9fd8-6d7e166055cb.PNG)

**Comment by bittusinghintruder** [comment]
@mjwheatley please have a look on the issue. I am stuck for the implementation. Due to this issue.
Thanks

**Comment by mjwheatley** [comment]
Please report the following:
compileSdkVersion
buildToolsVersion
minSdkVersion
targetSdkVersion

Do you have the support library added as a dependency in your build.gradle?
```
compile "com.android.support:support-v4:25.0.1"
```

**Comment by Richard6** [comment]
I have also encountered this problem, and how to solve it?

**Comment by Richard6** [comment]
@ @bittusinghintruder  , have you solved it?

**Comment by pareta2** [comment]
I am also facing the same problem any solutions yet?

**Comment by bittusinghintruder** [comment]
Sorry for the late reply @Richard6 . Not yet i am able to solve the problem in windows. But when i tried the same thing in MAC it works. Don't why but i have the same SDK installed on my both the systems. Really strange. 
@mjwheatley compile "com.android.support:support-v4:25.0.1" i have this  support library added as a dependency in my build.gradle. But still the problem persist in my windows. Even i tried by re installing my java and android SDK again.

**Comment by mjwheatley** [comment]
See if you can get this sample project to run https://developer.android.com/samples/FingerprintDialog/index.html

Then compare your project's settings with it.

**Comment by mjwheatley** [comment]
@bittusinghintruder have you had any success?
Please try installing again using the latest version I just published v1.3.0
I have removed the plugin dependency on support-v4 library.
If you are still getting errors that say 
```
android.hardware.fingerprint does not exist
import android.hardware.fingerprint.FingerprintManager;
```
then you just don't have the compile tools configured correctly for your project.

**Comment by DanceSC** [comment]
I am having the same issue, after adding the "cordova-plugin-android-fingerprint-auth" plugin.
I updated the gradle classpath to gradle:2.3.2 and it prompted me with 
" we strongly recommend that you update the Android Gradle plugin to version 3.0.1 and Gradle to version 4.1."
I clicked "Update" and this is what I have:

File->Settings->Build,Execution,Deployment->Gradle is set to "Use default gradle wrapper (recommended)"
File->ProjectStructure->Project
Gradle version: 4.1
Android Plugin Version: (empty)

On build.gradle (Module: android):
dependencies {
    classpath 'com.android.tools.build:gradle:2.3.2'
 }
task wrapper(type: Wrapper) {
    gradleVersion = '2.3.2'
}

When I click Run these are just some of the errors that I get:
Error:(19, 36) error: package android.hardware.fingerprint does not exist
Error:(21, 33) error: package android.security.keystore does not exist
Error:(22, 33) error: package android.security.keystore does not exist
Error:(71, 13) error: cannot find symbol class FingerprintManager
Error:(581, 58) error: package FingerprintManager does not exist

Any advise?
